src/Raven.Studio/typescript/viewmodels/resources/widgets/abstractDatabaseAndNodeAwareTableWidget.ts

```diff
@@ -171,14 +172,20 @@ abstract class abstractDatabaseAndNodeAwareTableWidget<TRaw, TStats extends stat
         });
 
         this.sortGridData(items);
-
+        
+        this.manageItems(items);
```

Comment:
items = this.manageItems(items);

and in default impl just return items. 

It will be easier to create new array in needed in future. 

Should we call that before or after sort?

Reply:
done
yes - need to call after sort..